snuba/snuba_migrations/profile_chunks/0001_create_profile_chunks_table.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime64, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.PROFILE_CHUNKS
+table_prefix = "profile_chunks"
+local_table_name = f"{table_prefix}_local"
+dist_table_name = f"{table_prefix}_dist"
+
+UNKNOWN_SPAN_STATUS = 2
+
+columns: List[Column[Modifiers]] = [
+    Column("project_id", UInt(64)),
+    Column("profiler_id", UUID()),
+    Column("chunk_id", UUID()),
+    Column(
+        "start_timestamp",
+        DateTime64(
+            precision=6,
+            modifiers=Modifiers(codecs=["DoubleDelta"]),
+        ),
+    ),
+    Column(
+        "end_timestamp",
+        DateTime64(
+            precision=6,
+            modifiers=Modifiers(codecs=["DoubleDelta"]),
+        ),
+    ),
+    Column("retention_days", UInt(16)),
+    Column("partition", UInt(16)),
+    Column("offset", UInt(64)),
+    Column("deleted", UInt(8)),
```

Comment:
Q.Q.: what's `deleted` for?

Reply:
For data retention reason. If a customer wants us to delete its data, we mark it as deleted.